Makefile

```diff
@@ -100,8 +100,8 @@ clean:
 
 .PHONY: \
 	all \
-	tools \
-	man \
+	$(TOOLS) \
+	$(MAN) \
```

Comment:
> [When you change the contents of the code, you want to re-generate the executable command.][1]

This is because we do not correctly list the dependencies for the `$(TOOLS)` (since calculating which Go source they depend on is not worth the trouble).  The fix should be to add `$(TOOLS)` to `.PHONY` and always rebuild (but don't remove `tools`).  The man-pages, on the other hand, *do* [correctly list their dependencies][3], so you don't need to add `$(MAN)`.

[1]: https://github.com/opencontainers/image-tools/pull/113#issuecomment-278872090
[2]: https://github.com/opencontainers/image-tools/blob/a358e03fde4e3628bf9fb7656bf643b63f975636/Makefile#L38-L39
[3]: https://github.com/opencontainers/image-tools/blob/a358e03fde4e3628bf9fb7656bf643b63f975636/Makefile#L42